### ISSUE TITLE
Trying to change a diskless drbd instance to plain fails

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1092]
Originally reported of Google Code with ID 1036.

```
What software version are you running? Please provide the output of "gnt-
cluster --version", "gnt-cluster version", and "hspace --version".

# gnt-cluster --version
gnt-cluster (ganeti v2.10.1) 2.10.1

# gnt-cluster version
Software version: 2.10.1
Internode protocol: 2100000
Configuration format: 2100000
OS api version: 20
Export interface: 0
VCS version: (ganeti) version v2.10.1

# hspace --version
hspace (ganeti) version v2.10.1
compiled with ghc 7.4
running on linux x86_64

<b>What distribution are you using?</b>

Ubuntu Server 12.04

<b>What steps will reproduce the problem?</b>
Instance 'instance.fqdn.com' is a drbd disk instance (status: ADMIN_down)
1. gnt-instance modify --disk remove instance.fqdn.com 
2. gnt-instance modify -t plain --ignore-ipolicy instance.fqdn.com

i.e. Trying to change a diskless instance's disk template from drbd to plain fails.

<b>What is the expected output? What do you see instead?</b>
Expected output: Disk template changed.
Actual output:
# gnt-instance modify -t plain --ignore-ipolicy instance.fqdn.com                                                                                                                
Failure: command execution error:
Internal assertion error: please report this as a bug.
Error message: ''; location:
  File "/usr/share/ganeti/2.10/ganeti/cmdlib/instance.py", line 3269, in _ConvertDrbdToPlain
    assert len(self.instance.secondary_nodes) == 1

<b>Please provide any additional information below.</b>
N/A
```
Originally added on 2015-03-06 12:26:00 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2015-03-16 14:37:46 +0000 UTC.

Added Labels: Type-Defect Priority-Medium
Changed State: Started
Added to Milestone: Release2.10

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed in commit:

commit 1ae08cbd28bafb4abda6dbf8fe79bff4b34e463e
Author: Hrvoje Ribicic <riba@google.com>
Date:   Mon Mar 16 22:54:04 2015 +0000

    Allow plain/DRBD conversions regardless of lack of disks
    
    Because of trivial issues, it was impossible to use standard mechanisms
    to convert a diskless plain instance to DRBD and vice versa. This
    patch fixes that with a mechanism which will work until later versions,
    where instances without disks organically take on the diskless
    template.
    
    Signed-off-by: Hrvoje Ribicic <riba@google.com>
    Reviewed-by: Helga Velroyen <helgav@google.com>

```
Originally added on 2015-03-18 13:13:18 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.14.0
```
Originally added on 2015-06-02 10:40:20 +0000 UTC.

Changed State: Released
